### PR TITLE
Issue 85

### PR DESCRIPTION
**Pull request by lkitching**

### CODE REVIEW

**Comment by Robsteranium** [comment]
I absent-mindedly rebased this branch locally before merging to master in 5d2febc. Now that all the commits have new shas, this branch isn't recognised by github as having been merged. I've tried pulling/ pushing my local change version (redundantly duplicating the commits with new shas) but it still appears as unmerged. Although the content itself is merged, the commits f356b78, 0345b45 and 41b586a are not.

Long story short, I'm closing this PR, although the branch itself has actually been merged.
